packages/zipkin/package.json

```diff
@@ -18,22 +18,5 @@
   "dependencies": {
     "base64-js": "^1.1.2",
     "is-promise": "^2.1.0"
-  },
-  "devDependencies": {
```

Comment:
there's a ton of redundancy in our package.json files, which I'll clean up as I can